lib/fli/FliImpl.cpp

```diff
@@ -243,59 +243,78 @@ GpiObjHdl* FliImpl::native_check_create(void *raw_hdl, GpiObjHdl *parent)
  */
 GpiObjHdl*  FliImpl::native_check_create(std::string &name, GpiObjHdl *parent)
 {
-    std::string fq_name = parent->get_fullname();
+    GpiObjHdl *parent_hdl = sim_to_hdl<GpiObjHdl *>(parent);
```

Comment:
Sorry about that, I'm not sure why but it is "wrong" in the code you used as a reference so my bad. sim_to_hdl was only intended to be used in the Gpi Layer, not that anyone would know that based on the name. 

If you are happy to remove them go ahead.